docs/usage.md

```diff
@@ -177,152 +226,193 @@ The following is an example `locationUpdate` event received by subscribers when
+### Track an individual location or member
 
-To stop subscribing to location updates, users can call the `locations.off()` method.
+To listen only to events for a specific member or location, we can use `createTracker`. This method filters messages based on a user provided predicate.
 
-### Track an individual location or user
+_Note that this is client-side filtering only, messages are still sent/received for updates._
 
-It is also possible to track a specific location or user, rather than subscribing to all events using `createTracker()`. All events will still be streamed to the client, however they will be filtered client-side.
-
-The following is an example of creating a tracker for a specific user based on their `clientId`, and then subscribing to updates for only that user:
+Create a tracker for a specific member based on their `clientId`, and then listen to updates for only that user:
 
 ```ts
-const memberTracker = space.locations.createTracker((change) => change.member.clientId === 'clemons#142');
+const memberTracker = space.locations.createTracker(
+  (locationUpdate) => locationUpdate.member.clientId === 'clemons#142',
+);
 
-memberTracker.on((change) => {
-  console.log(change);
+memberTracker.on((locationUpdate) => {
+  console.log(locationUpdate);
 });
 ```
 
-The following is an example of creating a tracker for a specific location, such as a single UI element, and then subscribing to updates for only that location:
+Create a tracker for a specific location, such as a single UI element, and listen to updates:
 
 ```ts
-const locationTracker = space.locations.createTracker((change) => change.previousLocation === 'slide-title');
+const locationTracker = space.locations.createTracker(
+  (locationUpdate) =>
+    locationUpdate.previousLocation === 'slide-title' || locationUpdate.currentLocation === 'slide-title',
+);
 
-locationTracker.on((change) => {
-  console.log(change);
+locationTracker.on((locationUpdate) => {
+  console.log(locationUpdate);
 });
 ```
 
-## Set a location
+## Live Cursors
+
+A common feature of collaborative apps is to show where a users cursors is positioned in realtime. It's easy to accomplish this with `cursors` API.
+
+Unlike a location, you can have multiple cursors. This can be used to represent multiple devices interacting with the UI or different ways of interacting with the browser (like scrolling).
 
-Users call `locations.set()` to publish a `locationUpdate` that will be received by all users subscribed to location updates. This should be called to update their location when they change position, or select a new UI element.
+The most common use case is however to show the current mouse/touchpad position.
 
-The following is an example of setting a location update:
+To get started, you'll need to get a named cursor instance:
 
 ```ts
-space.locations.set({ slide: '3', component: 'slide-title' });
+const mouseMove = space.cursors.get('mouseMove');
```

Comment:
Can we please have a different name for the cursor instance in this example? When we looked at it with Mark, it was confusing to understand which part is an Ably API and which is the DOM `mousemove` event.

Reply:
what would you suggest please?

Reply:
Let's call it `slidedeck-cursors`